src/main/java/workSpringData/Main.java

```diff
@@ -0,0 +1,22 @@
+package workSpringData;
+
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import workSpringData.repository.CustomersRepository;
+import workSpringData.service.CustomersService;
+import workSpringData.service.CustomersServiceImpl;
+
+public class Main {
+    public static void main(String[] args) {
+        ConfigurableApplicationContext context = new AnnotationConfigApplicationContext("workSpringData");
+
+        CustomersRepository customersRepository = context.getBean(CustomersRepository.class);
+        customersRepository.findAll().forEach(System.out::println);
```

Comment:
You should use CustomersService, not CustomersRepository.

Reply:
It depends to you.

Reply:
yes, I use it below. Repository was used just to try that it works))